multi-connector/src/main/scala/it/agilelab/darwin/connector/multi/MultiConnectorCreator.scala

```diff
@@ -0,0 +1,62 @@
+package it.agilelab.darwin.connector.multi
+
+import com.typesafe.config.Config
+import it.agilelab.darwin.common.{ Connector, ConnectorCreator, ConnectorFactory }
+import it.agilelab.darwin.manager.exception.DarwinException
+
+import it.agilelab.darwin.common.compat._
+
+object MultiConnectorCreator {
+  val REGISTRATOR                      = "registrator"
+  val CONFLUENT_SINGLE_OBJECT_ENCODING = "confluent-single-object-encoding"
+  val STANDARD_SINGLE_OBJECT_ENCODING  = "standard-single-object-encoding"
+}
+class MultiConnectorCreator extends ConnectorCreator {
+
+  /**
+    * @return the name of the Connector
+    */
+  override def name(): String = "multi"
+
+  private def mergeConf(conf: Config, path: String): Config = {
+
+    conf
+      .getConfig(path)
```

Comment:
I think here we could have some problems if we don't specify clearly what the user should expect from the configuration. Honestly, I don't recall why we are adding the level-specific configurations to the upper level instead of just doing `conf.getConfig(path)`: are there some 1st-level configurations that need to be passed to the specific creators that are not defined in the lower connector-specific levels?
Or in other words, we are saying that the sub-configuration of a connector could be incomplete and needs something from the upper level (like default values shared by all the connectors)?

Reply:
I see no harm

Reply:
yes, I agree with you that there is no harm, but it should be clear for users what they should do when defining the configuration (maybe adding a note in the readme.md?).
Just to say: if you write a configuration at upper level it will be passed to all the connectors, and if you define a key inside a connector that is already defined at common level, the specific one will overwrite the general one.